src/BenchmarkDotNet/Toolchains/Executor.cs

```diff
@@ -116,7 +116,10 @@ private ProcessStartInfo CreateStartInfo(BenchmarkCase benchmarkCase, ArtifactsP
                 case WasmRuntime wasm:
                     start.FileName = wasm.JavaScriptEngine;
                     start.RedirectStandardInput = false;
-                    start.Arguments = $"{wasm.JavaScriptEngineArguments} main.js -- --run {artifactsPaths.ProgramName}.dll {args} ";
+
+                    string main_js = (runtime.RuntimeMoniker == RuntimeMoniker.WasmNet60 || runtime.RuntimeMoniker == RuntimeMoniker.WasmNet50)? "main.js" : "test-main.js";
```

Comment:
the condition can be simplified

```suggestion
                    string main_js = runtime.RuntimeMoniker < RuntimeMoniker.WasmNet70 ? "main.js" : "test-main.js";
```

thanks for taking care of backward compatibility! 👍